tests/boardfarm_plugins/boardfarm_prplmesh/prplmesh_config.json

```diff
@@ -0,0 +1,9 @@
+{
+    "prplmesh_docker": {
+        "name": "dockerized_device",
+        "board_type": "prplmesh_docker",
+        "role": "controller",
+        "conn_cmd": "",
```

Comment:
I believe this is only needed for DebianBox, not for LinuxDevice...

Reply:
> I believe this is only needed for DebianBox, not for LinuxDevice...

Unfortunately - no.
Field `conn_cmd` is mandatory as it has no default value.
There's no dependency on device class, it's only boardfarm parser's issue.
When boardfarm attempts to parse custom JSON config without that field - exception will be thrown.